src/styles/blocks/body.scss

```diff
@@ -0,0 +1,13 @@
+.body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
```

Comment:
Move font styles to the html selector, and `font-style: normal` is default value, so you need to remove it
```suggestion
  font-family: Roboto, sans-serif;
  font-weight: 500;
  font-size: 12px;
```

Reply:
+